rdkcentral/Lightning-UI-Components
# fix: lock lng core version to avoid bug introduced in 2.12.0

## Description
Upgrading `@lightningjs/core` to version `2.12.0` results in multiple unit tests timing out and failing in `Artwork.test.js`. I believe this change in `@lightningjs/core` is what resulted in these test failures: https://github.com/rdkcentral/Lightning/pull/508.

The failing Artwork unit tests are all awaiting a method that gets triggered from the `txLoaded` event, which seems to now be emitted less times due to the above change. We'll lock the `@lightningjs/core` in this repo to `2.11.0` until we've resolved the issues.


## Testing
1. delete `yarn.lock` file and do a fresh install with the  `yarn` command
2. run the unit testing suite: `yarn workspace @lightningjs/ui-components test`
3. All tests should pass. (If a `^` is prepended to the `@lightningjs/core` version in `package.json` files and these steps are repeated, version 2.12.0 will be installed and the Artwork tests will fail)


## Checklist
